src/inject/dynamic-theme/stylesheet-proxy-mv3-injector.ts

```diff
@@ -0,0 +1,11 @@
+import {injectProxy} from './stylesheet-proxy';
```

Comment:
The file name seems long. Maybe it can be shortened like `mv3-proxy.ts` or similar?
![4xk](https://user-images.githubusercontent.com/9638588/175659964-aa0a2005-e6fb-4486-ab02-111e77fc7e8d.gif)